subprojects/test-runner/test-report/src/main/kotlin/com/avito/android/test/report/video/VideoCapturer.kt

```diff
@@ -37,29 +37,18 @@ class VideoCapturerImpl(
     override fun start(): Result<Unit> {
         // checks if execute start() concurrently
         return if (state is State.Idling) {
-
-            val (videoFile, videoError) = testArtifactsProvider.generateUniqueFile("mp4")
-
-            if (videoError != null) {
```

Comment:
for example, we check that the video error is not null so assume that the video file is null
It pushed us to use force unwrap